### ISSUE TITLE
MeasureObjectIntensityDistribution help menu couldn't start

### ISSUE THREAD

**Issue by minh-doan** [CellProfiler/CellProfiler#3344]
Clicking "?" for help menu brings this error:

```
Traceback (most recent call last):
File "/Users/minhdoan/CellProfiler/CellProfiler/cellprofiler/gui/cpframe.py", line 972, in __on_help_module
self.do_help_modules(modules)
File "/Users/minhdoan/CellProfiler/CellProfiler/cellprofiler/gui/cpframe.py", line 1004, in do_help_modules
self.do_help_module(module.module_name, module.get_help())
File "/Users/minhdoan/CellProfiler/CellProfiler/cellprofiler/module.py", line 255, in get_help
"MODULE_DOC": self._rst_to_html_fragment(self.__doc__),
File "/Users/minhdoan/CellProfiler/CellProfiler/cellprofiler/module.py", line 212, in _rst_to_html_fragment
parts = docutils.core.publish_parts(source=source, writer_name="html")
File "/usr/local/lib/python2.7/site-packages/docutils/core.py", line 450, in publish_parts
enable_exit_status=enable_exit_status)
File "/usr/local/lib/python2.7/site-packages/docutils/core.py", line 664, in publish_programmatically
output = pub.publish(enable_exit_status=enable_exit_status)
File "/usr/local/lib/python2.7/site-packages/docutils/core.py", line 217, in publish
self.settings)
File "/usr/local/lib/python2.7/site-packages/docutils/readers/__init__.py", line 71, in read
self.input = self.source.read()
File "/usr/local/lib/python2.7/site-packages/docutils/io.py", line 435, in read
return self.decode(self.source)
File "/usr/local/lib/python2.7/site-packages/docutils/io.py", line 99, in decode
data_encoding = self.determine_encoding_from_data(data)
File "/usr/local/lib/python2.7/site-packages/docutils/io.py", line 142, in determine_encoding_from_data
if data.startswith(start_bytes):
AttributeError: 'NoneType' object has no attribute 'startswith'
```

**Comment by bethac07** [comment]
Verified in Windows release.

**Comment by AetherUnbound** [comment]
I believe this is related to #3407 (seeing as `MeasureObjectIntensityDistribution` was the module causing the error).

**Comment by mcquin** [comment]
Resolved #3408
